lib.php

```diff
@@ -161,9 +168,15 @@ public function embed($urls, $name, $width, $height, $options) {
             if (isset($options['subtitles'])) {
                 $tracks = array();
                 foreach ($options['subtitles'] as $label => $subtitlefileurl) {
-                    $tracks[] = array(
-                        'file' => $subtitlefileurl->out(),
-                        'label' => $label);
+                    if ($label == 'chapters' || $label == 'thumbnails') {
+                        $tracks[] = array(
+                            'file' => $subtitlefileurl->out(),
```

Comment:
As the URL will be embedded in the Javascript code we need the unescaped version. Otherwise jwplayer could not parse the html escaped URL. 

You dropped the subtitle feature by adding the if statement.